### DISCUSSION TITLE
How can I make children not hoverable except the parent div ?

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3611] [Q&A] [answered]
Hello @artf,
What I want is all the direct children of body tag should be hoverable but want to disable this feature inside of children of particular section tag
```
<section>
  <div> hello world </div>
   <div> hello artf </div>
</section>
```
section tag should be hoverable but want to disable for nested divs.
```
editor.on('component:selected', cmp => {
 cmp.set({
   hoverable: false,
   selectable: false,
   propagate:['hoverable,''selectable']
 })

}
```
This doesnot work

**Comment by artf** [comment, marked as answer]
Probably the easiest way would be adding a custom style to your section component which disables pointer-events to the inner elements

**Reply by iabhiyaan** [reply]
Thankyou @artf
